### DISCUSSION TITLE
Jump to bottom Add Web Follower Setting for Custom "Next poll in" Frequency

### DISCUSSION THREAD

**Discussion by gilla013** [NightscoutFoundation/xDrip#2187] [Ideas]
Next poll in is always 5 min and can not be set lower. Though w/ my libre 3 I believe lowering this would result in a shorter reading interval which also displays on my watch.
If I am wrong about this sorry, but if this is correct, please add this as a parameter in the Web Follower Settings.

I guess more FSL3 users would like this.

**Comment by Navid200** [comment]
@keencave

**Comment by PythonFZ** [comment]
The WebFollower feature is great! Xdrip supporting the 1 min intervall would be a great addition. If tests are required I am happy to help.

**Comment by Navid200** [comment]
I've just been told the web follower interval is planned to stay at 5 minutes.

**Reply by gilla013** [reply]
Missed chance unless we can get another hardware source that makes intervals shorter. 5 min for the FSL3 is just ridiculous.

**Reply by PythonFZ** [reply]
Is there a reasoning for this decission? I'd assume that in the future more vendors will go for a 1 min intervall and this will come up again eventually.

**Reply by gilla013** [reply]
An alternative just for FSL3 would be using locally installed libre 3 or librelinkup app from the smartphone as new hardware source. But I agree that applying this (as an option) to the web follower would be applicable to multiple vendor usages.

**Reply by Navid200** [reply]
I know that OOP raw or something like that show at a 1-minute interval.
I am not sure how we have accomplished that considering there is a lot inside xDrip that has been built around a 5-minute interval.

This capability, web follower, was added just recently.
There is a lot more that needs to happen in xDrip.  Just look at the list of open PRs.

Anyway, I have communicated your concerns with the developers and have told you all I have been told.  I have again shared your feedback.  We can now only wait.

**Reply by gilla013** [reply]
Thanks for your response

**Reply by salzmret** [reply]
I think there is a missunderstanding.. they asked for the raw values (every 1 minute). See libre 2 or libre 3 with llu client.. it would be great if the web follower will show the raw values too.. that doesnt change the 5min interval..

**Comment by DieSchatten** [comment]
Hi,

the LLU Web Follower is not working here in Germany for Libre. Seems like Abbott did change the URL at least for German Accounts from api-eu.libreview.io to api-de.libreview.io . Is there a way to change that?

**Reply by jamorham** [reply]
Which version are you using? Please try the latest nightly to see if it resolves the problem.

**Reply by jamorham** [reply]
Please also try togging the proxy support on then off again as this refreshes the internal state.

**Reply by DieSchatten** [reply]
It does not seem to work. 

Do I need to manually change some settings or just leave it in default?

**Reply by jamorham** [reply]
Did it previously work for you?

**Reply by DieSchatten** [reply]
Yes it did work fine, until around 2 weeks ago. In a German diabetic forum I did read that Abbott changed the URL of some servers to LibreLinkUp. I assume thats the reason it is not working anymore.
I am using this  app at the moment which is working, but uses alot more battery than xdrip.
https://mega.nz/file/h5UiRIAI#_coNr1qxSzIeF24HIXabVvYoAgBAno5uag_Lu9yHMjY
https://insulinclub.de/index.php?thread/33987-llu-client/&pageNo=5

**Reply by DieSchatten** [reply]
It seems to work now. I guess something was wrong on my end. I did reset all and installed the latest nightly, now it works. Sorry for the trouble.

**Comment by gui-dos** [comment]
You can get the current regional server for a specific country and a list of all the other subdomains just by scraping for example:


[https://api.libreview.io/llu/config/country?country=DE](https://api.libreview.io/llu/config/country?country=DE)

**Comment by gilla013** [comment]
Raw data on libreview is 5 min by default, but every time you open librelinkup which is the source for the data that the web follower uses you get the current value, directly from libre 3 which is accurate by 60 sec.
Why not change the web follower or create a new hardware source, that makes a call to the librelinkup server on demand(or by waking up watch screen) or every minute, so we can have more accurate reading on our watch.

It is such a missed opportunity to continue 5 minute readings on a FSL3 while we can have 1 min. It doesn't make sense.

**Reply by salzmret** [reply]
I think the 5 min interval in combination with the 1min raw values is the best.. same as libre 2..

**Reply by gilla013** [reply]
Everything that can make me use full advantage of my libre 3 will do =)

**Comment by gilla013** [comment]
Any progress on a shorter interval? 
I  managed to create a job queue that polls the following API every min: https://api-eu.libreview.io/llu/connections/{patienId]/graph 
Using the value glucoseMeasurement.Value I can get my current glucose level every minute. This gives me the idea that it's possible for xDrip aswell. Can you use direct API calls in stead of the LibreView data?
![per min](https://user-images.githubusercontent.com/43918803/190242049-72a5d9dc-9987-497f-9a79-e747684eb25a.png)
![cgm data](https://user-images.githubusercontent.com/43918803/190242051-83e1c129-d3dd-4e23-ab2c-baa67e240dd8.jpg)

**Comment by iicd2386cd** [comment]
I have a problem because I get data every 10 minutes. I set web fallower, it downloads data every 10 minutes and I don't know what could be the reason :( any hint?

**Comment by iicd2386cd** [comment]
I have a problem because I get data every 10 minutes. I set web fallower, it downloads data every 10 minutes and I don't know what could be the reason :( any hint?
